ui/src/app/applications/components/utils.tsx

```diff
@@ -491,7 +491,7 @@ function getActionItems(
         })
         .catch(() => [] as MenuItem[]);
 
-    const resourceActions = getResourceActionsMenuItems(resource, application, appContext);
```

Comment:
@rbreeze shouldn't a mistake like this one be caught by the type checks?

Reply:
You're right, it should... not sure why it didn't this time.